### PR TITLE
docs(ja): translate guides/features/rendering-modes

### PR DESCRIPTION
**Pull request by szk18**
@aytdm @inouetakuya @53able 
I translated `features/rendering-modes.md`. please review this PR✨

**Ref**
- https://github.com/nuxt/nuxtjs.org/issues/544
- https://github.com/vuejs-jp/ja.nuxtjs.org/issues/22

### CODE REVIEW

File: content/ja/guides/features/rendering-modes.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+category: features
+---
+
+## ユニバーサル
+
+`mode: 'universal'`: アイソモーフィックなアプリケーション (サーバーサイドレンダリング、または静的なサイト)
+
+```js{}[nuxt.config.js]
+export default {
+  mode: 'universal' // default universal
```
Comment (aytdm) [inline content/ja/guides/features/rendering-modes.md:14]: コメントの翻訳もお願いします

```suggestion
  mode: 'universal' // mode の default 値はユニバーサル
```

**Review by aytdm — COMMENTED** [review]
Thank you for your prompt translating! I commented.

File: content/ja/guides/features/rendering-modes.md (hunk 1 above)
Comment (szk18) [reply content/ja/guides/features/rendering-modes.md]: コメントありがとうございます
https://github.com/nuxt/nuxtjs.org/pull/616/commits/c5a80e5331d2eece78d7917afda85fe6db76120d で修正しました。

Comment (aytdm) [reply content/ja/guides/features/rendering-modes.md]: ありがとうございます！

**Review by aytdm — APPROVED** [review]
Thank you 😊

**Comment by aytdm** [comment]
@debs-obrien I tried to make you a reviewer, but couldn't. I would like you to merge this PR when you are available. Thank you.

**Comment by debs-obrien** [comment]
no worries. just tagging me will work for now. thanks

**Comment by aytdm** [comment]
The first Japanese translation of "guides" 🎉

**Pull request merged**